index.js

```diff
@@ -818,7 +847,7 @@ function _applyFragment (fragment) {
       })[0]
 
       if (!stateName) return // fragment contains a nonexistent state
-      state = states[stateName]
+      var state = states[stateName]
```

Comment:
Can you use [strict mode](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode) in this file?

Reply:
Yeah, that makes sense. (b95cc5a)